doc/user_guide/getting_going.rst

```diff
@@ -359,14 +359,14 @@ supplied then the script writes the generated/re-written Fortran to
 the terminal. For details of the other command-line arguments please
 see the :ref:`psyclone_script` Section.
 
-Examples are provided in the examples directory of the PSyclone git
+Examples are provided in the ``examples`` directory of the PSyclone git
 repository - if you have cloned the repository then ``EGS_HOME`` in
 what follows is the root ``PSyclone`` directory. Alternatively, if you
 have installed PSyclone using pip then they may be found in the
 ``share/psyclone`` directory under your Python installation. In this
-case you should copy them to some convenient location (hereafter
-called ``EGS_HOME``) before attempting to carry out the following
-instructions.
+case you should copy the whole ``examples`` directory to some convenient
+location (hereafter called ``EGS_HOME``) before attempting to carry out
+the following instructions.
```

Comment:
While now all the required files are indeed copied when an install is done, I still cannot get the examples to work after a normal (not `-e`) installation: If I follow the instructions and copy the examples into a separate directory, the makefile adds a psyclone command line option for the config file, e.g.:
```
joerg@localhost:~/xxexamples/gocean/eg5> export PSYCLONE_CONFIG=/home/joerg/work/psyclone/config/psyclone.cfg;  make transform
psyclone --config /home/joerg/config/psyclone.cfg --profile invokes -l -api "gocean1.0" \
             -opsy psy.f90 -oalg alg.f90 test.x90
```
An explicit --config argument to a non-existing file aborts reading of the config file, it does not fall back to $PSYCLONE_CONFIG (which I think might be the right thing to do). Not sure what's the best way to fix this ... let the makefile test if PSYCLONE_CONFIG is set, and if so, don't add --config?

If additional steps are required after copying the examples into a special directory to make the examples run (e.g. setting PSYCLONE_CONFIG), they should also be documented.

Reply:
I had it in my head that using `make` wouldn't be something that the average user would do. It's covered in the dedicated Examples section but not in Getting Going (which does not mention `make`). The fix (specified in the Examples section) is to set $PSYCLONE before running make. Possibly I could improve the Makefile so that it does not set --config if it can't find the config file. I'll take a look.

Reply:
I think I have something robust now. If the config file is not found (in `common.mk`) then no attempt is made to use the `--config` flag with `psyclone`. We then fall back to picking up the `PSYCLONE_CONFIG` environment variable. I've updated the documentation appropriately. I also discovered that one of the examples relied on a kernel in the `test_files/dynamo0p3` directory and therefore did not work if the examples directory was moved. I've copied the necessary kernel file into the example to make it self sufficient.